<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Prem" failures="1" tests="1" name="LoginTest_Chrome" time="7.646" errors="0" timestamp="2025-09-25T00:04:11 IST">
  <testcase classname="com.framework.tests.contactUsTestPage" name="verifyLogin" time="4.763">
    <failure type="java.lang.AssertionError" message="expected [Thank you for your inquiry! We will get back to you within 24 hours.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Thank you for your inquiry! We will get back to you within 24 hours.] but found []
at com.framework.pages.contactUsPage.dropUs(contactUsPage.java:53)
at com.framework.tests.contactUsTestPage.verifyLogin(contactUsTestPage.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- verifyLogin -->
</testsuite> <!-- LoginTest_Chrome -->
